File: abc/abc199/a.cpp
// https://atcoder.jp/contests/abc199/submissions/25620659

#include <bits/stdc++.h>
using namespace std;

int main() {
  int A, B, C;
  cin >> A >> B >> C;
  if(pow(A, 2) + pow(B, 2) < pow(C, 2)) cout << "Yes";
  else cout << "No";
}
  

File: abc/abc199/a.java
// https://atcoder.jp/contests/abc199/submissions/40777971

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);
    
    int a = stdIn.nextInt();
    int b = stdIn.nextInt();
    int c = stdIn.nextInt();

    if(a * a + b * b < c * c) System.out.println("Yes");
    else System.out.println("No");
  }
}
